src/Fable.Import.ReactNative.fs

```diff
@@ -1898,6 +1898,7 @@ module ReactNative =
         abstract setNetworkActivityIndicatorVisible: (bool -> unit) with get, set
         abstract setBackgroundColor: (string -> bool -> unit) with get, set
         abstract setTranslucent: (bool -> unit) with get, set
+        abstract currentHeight: float with get // Only available for Android
```

Comment:
Can you please rewrite the comment as:

```fs
/// Only available for Android
abstract currentHeight: float with get
```

I know for sure the `///` is needed so it's accessible when using the `.dll`. For the position, I am not sure but this where I always place it.